2017-02/feb-22.md

```diff
@@ -27,14 +30,21 @@ We've added a few folks as collaborators along the way!
 | github | [@aaronang](https://github.com/aaronang) | [@yavorsky](https://github.com/yavorsky) | [@STRML](https://github.com/STRML) | [@chicoxyzzy](https://github.com/chicoxyzzy) |
 | twitter | [@_aaronang](https://twitter.com/_aaronang) | [@yavorsky_](https://twitter.com/yavorsky_) | [@STRML_](https://twitter.com/STRML_) | [@chicoxyzzy](https://twitter.com/chicoxyzzy) |
 
-- Aaron just joined our slack this past week and as been super excited about getting involved: made a few PRs and efforts to improve our site and contributing docs.
+- Aaron just joined our slack this *past week* and has been super excited about getting involved: made a few PRs and efforts to improve our site and contributing docs.
 - Artem has been making a lot of great PRs to babel-preset-env and is also getting started with other stuff.
 - Samuel has made most of the PRs for fixing the bugs that we've having with the react constant elements plugin as well as added discourse comments support to our docs pages.
 - Sergey helps maintain compat-table and has been helping with babel-preset-env.
 
 ### Future Discussions
 
-> TODO
+> babel-preset-env and its future
+
+Should preset-env be in core? How can it be better integrated with babili/webpack?
```

Comment:
One of the things I loved about babel 6 (and people came around to it), is that babel became a platform for code transformation. It's no longer about ES-spec transformation (necessarily), but general code transformation. I can appreciate the desire to bring the preset-env into core (because most people will be using it), but I would discourage against it for this reason. Unless there's some strong compelling argument for it...